### ISSUE TITLE
Tooltip with additional information when hovering emoji

### ISSUE THREAD

**Issue by nolanlawson** [nolanlawson/emoji-picker-element#1]
Using `title` is the easiest option, but it has a few drawbacks:

- doesn't work for keyboard users
- can't display rich data, only text

I think there's a case to be made that we should add some kind of custom tooltip. It could show things like a larger version of the emoji, the shortcodes, the annotation, etc. Using a pattern like [this](https://github.com/nolanlawson/pinafore/blob/master/src/routes/_components/Tooltip.html), we could even make it accessible to keyboard and screen reader users.

The main downside is increasing the bundle size and adding complexity. Also either way, it's not going to be accessible to mobile touchscreen users (although this could be said of `title` as well).
